src/components/EditSnack/EditSnack.js

```diff
@@ -1,26 +1,55 @@
-import React, {Component} from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import ButtonList from '../ButtonList/ButtonList';
+import ListSelection from '../listSelection/ListSelection';
 import Logo from '../Logo/Logo';
+import './EditSnack.css';
 
-class EditSnack extends Component {
-	handleClick = (id, name) => {
-		this.props.setActualItem(id);
-		this.props.history.push('/slackname');
-	};
+const importImages = require.context('./assets', true, /\.svg$/);
+const imgFilesObject = importImages.keys().reduce((images, key) => {
+	images[key] = importImages(key);
+	return images;
+}, {});
 
-	render() {
-		return (
-			<div>
-				<Logo />
-				<ButtonList
-					items={this.props.items}
-					onClick={this.handleClick}
-				/>
+const EditSnack = props => {
+	const items = props.items.map(item => {
+		const relativeImagePath = './' + item.image;
+		let actualImagePath;
+
+		if (imgFilesObject[relativeImagePath]) {
+			actualImagePath =
+				'./' + item.image + (item.image.endsWith('.svg') ? '' : '.svg');
+		} else {
+			actualImagePath = './misc-bar.svg';
+		}
+
+		const image = importImages(actualImagePath);
+		return {
+			...item,
+			image
+		};
+	});
+	return (
+		<div>
+			<Logo />
+			<div className="textinfo">
```

Comment:
I don't think this is consistent elsewhere but it'd be good to start, the usual convention is to use kebab-case (so text-info).

It'd be good if we could stick to bem as well, so it'd end up looking something like `className="edit-snack edit-snack--text-info"`